language/move-lang/tests/move_check/naming/friend_decl_self.exp

```diff
@@ -5,8 +5,8 @@ error:
  3 │     friend Self;
    │     ^^^^^^^^^^^^ Invalid friend declaration
    ·
- 3 │     friend Self;
-   │            ---- Cannot declare the module itself as a friend
+ 2 │ module M {
+   │        - Cannot declare the module itself as a friend
```

Comment:
Yeah it's a bit tricky since this one happens after expansion. It is pointing to the source of the alias. You would get the same thing with 
```
module 0x42::M {
    use 0x42::M as M2; < Cannot declare the module itself as a friend
    friend M2; < Invalid friend decl
}
```
The others are not module name aliases so it points directly there

Reply:
These might get switched aroudn when I redo every error message. As you will get something like 

> Error cannot declare module itself as a friend 
> Invalid Friend Decl
> Module Resolved here

Or something

Reply:
OK, this one isn't a big deal